src/lib/providers.ts

```diff
@@ -28,6 +42,13 @@ const initializeTreeDataProviders = (context: ExtensionContext) => {
     TREE_DATA.HELP_AND_FEEDBACK,
     helpAndFeedbackProvider
   );
+  window.registerTreeDataProvider(TREE_DATA.REMOTE, remoteFsProvider);
+  context.subscriptions.push(
+    commands.registerCommand(COMMANDS.REMOTE_FS.REFRESH, () => {
+      console.log(COMMANDS.REMOTE_FS.REFRESH);
+      remoteFsProvider.refresh();
+    })
+  );
```

Comment:
We also register our `TreeDataProvider` (the remote fs view) and the command to refresh the remote fs view, which calls `refresh()` on said provider